src/entity/presentationWrapper/verifiableCredentialEntity.ts

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
this class is dangerous.. see my comment [here](https://sphereon.atlassian.net/browse/ESSIFI-55?focusedCommentId=34676)

Reply:
I didn't get it. Do you want me to wrap it inside another class? It's already inside the presentation which is wrapped by the presentation wrapper.